<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-HR9B3S3" failures="1" tests="4" name="PetTests" time="3.666" errors="0" timestamp="2024-08-06T14:19:26 EDT">
  <testcase classname="api.test.UserTests2" name="testPostUser" time="1.9"/>
  <testcase classname="api.test.UserTests2" name="testGetUserByName" time="0.249">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests2.testGetUserByName(UserTests2.java:63)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testGetUserByName -->
  <testcase classname="api.test.UserTests2" name="testUpdateUserByName" time="0.41"/>
  <testcase classname="api.test.UserTests2" name="testDeleteUserByName" time="0.18"/>
</testsuite> <!-- PetTests -->
